config/README.md

```diff
@@ -23,7 +23,7 @@ Common linting configuration utilizing:
 
 - [ESLint](https://eslint.org/)
 - [TypeScript ESLint](https://github.com/typescript-eslint/typescript-eslint)
-- [TypeStrict](https://github.com/krzkaczor/TypeStrict)
+- [Type-strict](https://github.com/krzkaczor/Type-strict)
```

Comment:
This is wrong

Reply:
Oof, this must be caused by a find-and-replace, good find

Reply:
Have reverted back to correct URL!